data/data/aws/cluster/main.tf

```diff
@@ -5,7 +5,15 @@ locals {
     },
     var.aws_extra_tags,
   )
-  description = "Created By OpenShift Installer"
+  description    = "Created By OpenShift Installer"
+  new_tag_len    = 8
+  sliced_tag_map = length(var.aws_extra_tags) <= local.new_tag_len ? var.aws_extra_tags : { for k in slice(keys(var.aws_extra_tags), 0, local.new_tag_len) : k => var.aws_extra_tags[k] }
+  s3_object_tags = merge(
+    {
+      "kubernetes.io/cluster/${var.cluster_id}" = "owned"
```

Comment:
Is the bucket successfully deleted by the cluster destroy? If so, it should be easy to add an optional ci job for this and we could have some observable testing in CI.

Reply:
Yeah it did get destroyed when I tested it. I'll create a step for this.